src/superstruct.ts

```diff
@@ -31,7 +20,7 @@ const parseErrorSchema = (
                       currentPath,
                       validateAllFieldCriteria,
                       previous,
-                      type,
+                      type!,
```

Comment:
@bluebill1049 `type` may be `undefined`. 
using `!` to ignore type errors, how should I fix this?

Reply:
maybe we should omit the type if it can be `undefined`? your thoughts?

Reply:
@bluebill1049 If we omit type, I think you need to fix `FieldError` type as follows.

```diff
 export type FieldError = {
-  type: string;
+  type?: string;
   ref?: Ref;
   types?: MultipleFieldErrors;
   message?: Message;
   isManual?: boolean;
 }
```

I think it's better to use an empty string (`''`) to avoid this fix.

Reply:
oh that's right, let's leave as ''

Reply:
I will fix this!